images/win/scripts/Installers/Install-PostgreSQL.ps1

```diff
@@ -0,0 +1,10 @@
+$ErrorActionPreference = "Stop"
+
+Import-Module -Name ImageHelpers
+
+choco install postgresql
+
+$paths=(Get-CimInstance Win32_Service -Filter "Name LIKE 'postgresql-%'").PathName
+$pgbin=$paths.split('"')[1].replace("\pg_ctl.exe", "")
+Add-MachinePathItem $pgbin
+$env:Path = Get-MachinePath
```

Comment:
Do we use it?

Reply:
Thank you, I have removed useless string